package.json

```diff
@@ -23,6 +23,16 @@
     "profile": "beefy tests/bench/bench.js --open -- -t brfs -t ./tests/bench/xform-rules.js -r espree",
     "coveralls": "cat ./coverage/lcov.info | coveralls"
   },
+  "gitHooks": {
+    "pre-commit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "node ./bin/eslint.js --fix",
+      "git add"
+    ],
+    "*.md": "markdownlint"
```

Comment:
Just to make sure I understand: We need `git add` in the `*.js` section because eslint --fix might make changes, so we need to stage those; but we don't need it here, because markdownlint doesn't autofix, but rather only reports errors.

Have I got that right?